pkg/scheduler/schedule_one_test.go

```diff
@@ -852,7 +852,11 @@ func TestSchedulerScheduleOne(t *testing.T) {
 				if diff := cmp.Diff(item.expectBind, gotBinding); diff != "" {
 					t.Errorf("got binding diff (-want, +got): %s", diff)
 				}
-				if len(queue.InFlightPods()) != 0 {
+				// We have to use wait here
+				// because if the Pod goes to the binding cycle, the inflight pods might be not empty immediately at this point.
```

Comment:
Consider saying "because if the Pod is still in the binding cycle" or "We have to wait here until all Pods finish binding".

Reply:
the suggested one is inappropriate a bit because not all the Pods go to the binding cycle. But, I modified the comment anyways.